website/docs/r/emr_cluster.html.markdown

```diff
@@ -282,7 +284,8 @@ Supported arguments for the `core_instance_group` configuration block:
 
 * `instance_type` - (Required) EC2 instance type for all instances in the instance group.
 * `autoscaling_policy` - (Optional) String containing the [EMR Auto Scaling Policy](https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-automatic-scaling.html) JSON.
-* `bid_price` - (Optional) Bid price for each EC2 instance in the instance group, expressed in USD. By setting this attribute, the instance group is being declared as a Spot Instance, and will implicitly create a Spot request. Leave this blank to use On-Demand Instances.
+* `market` - (Required) The instance purchasing option. Valid values are ON_DEMAND or SPOT. 
+* `bid_price` - (Optional) The maximum Spot price your are willing to pay for EC2 instances. An optional, nullable field that applies if the MarketType for the instance group is specified as SPOT. Specify the maximum spot price in USD. If the value is NULL and SPOT is specified, the maximum Spot price is set equal to the On-Demand price.
```

Comment:
We should make this documentation more operator friendly, unless `NULL` is actually a valid value.

```suggestion
* `bid_price` - (Optional) If set, the bid price for each EC2 instance in the instance group, expressed in USD. By setting this attribute, the instance group is being declared as a Spot Instance and will implicitly create a Spot request. If not set while `market` is configured to `SPOT`, the maximum price is set equal to the On-Demand price.
```

Similar changes to `master_instance_group` below.